pkg/urbit/include/vere/vere.h

```diff
@@ -156,6 +156,7 @@
           c3_o    mou;                      //  M (for mouse event) received
           c3_y    ton_y;                    //  mouse button
           c3_y    col_y;                    //  column coordinate
+          c3_y    seq;                      //  vt sequence
```

Comment:
Oops, forgot to include this in the previous review. We always include the size of the integer in their variable names.

```suggestion
          c3_y    seq_y;                    //  vt sequence
```

Reply:
Ah good catch, I've fixed it up.